themes/socialbase/templates/node/node--teaser.html.twig

```diff
@@ -80,15 +80,30 @@
   ]
 %}
 
-<div {{ attributes.addClass(classes) }}>
+<div{{ attributes.addClass(classes) }}>
 
   <div class='teaser__image'>
 
     {% if view_mode == 'teaser' %}
-      {% block card_teaser_type %} {% endblock %}
+      {% block card_teaser_type %}
+
+        <div class="teaser__teaser-type">
+          <svg class="teaser__teaser-type-icon">
+            <use xlink:href="#icon-{{- node.bundle|clean_class -}}"></use>
+          </svg>
+        </div>
+
+      {% endblock %}
     {% endif %}
 
-    {% block card_image %} {% endblock %}
+    {# allows for new content types to just use the plain field_image name #}
+    {% block card_image %}
+      {{ content.field_event_image }}
+      {{ content.field_topic_image }}
+      {{ content.field_page_image }}
+      {{ content.field_book_image }}
+      {{ content.field_image }}
```

Comment:
It won't work if I have named my image differently correct? E.g. `field_different_image`.

This will solve part of the problem for sure, but I can imagine we might need to make it more robust still. What do you think @MaikelGG?